Assets/Countly/Plugins/CountlySDK/Interfaces/IViewModule.cs

```diff
@@ -1,26 +1,126 @@
 using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-
-// interface for SDK users
-public interface IViewModule
-{
-    /// <summary>
-    /// Start tracking a view
-    /// </summary>
-    /// <param name="name">name of the view</param>
-    /// <returns></returns>
-    [Obsolete("RecordOpenViewAsync(string name, IDictionary<string, object> segmentation = null) is deprecated, this is going to be removed in the future.")]
-    Task RecordOpenViewAsync(string name, IDictionary<string, object> segmentation = null);
-
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+// interface for SDK users
+public interface IViewModule
+{
+    /// <summary>
+    /// Starts a view which would not close automatically
```

Comment:
if app will crash or close all views should be closed automatically. what you wanted to say here is more close to 'starts a view with the given name. You can use it to open multiple views in parallel.'